dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/persistence/file/FileOrchestrator.kt

```diff
@@ -21,11 +21,13 @@ import java.io.File
 internal interface FileOrchestrator {
 
     /**
+     * @param forceNewBatch if `true` will force the orchestrator to start a new file.
```

Comment:
Mismatch between name in the doc and name in the function signature. I suppose it should be `forceNewFile` here?